strings.txt

```diff
@@ -31443,20 +31446,30 @@ fa = دوباره نپرس
    ru = Отчеты об ошибках
 
  [opt_out_fabric_description]
-   en = MAPS.ME uses your data to improve your experience.
-   ru = MAPS.ME использует ваши данные, чтобы улучшать приложение.
+   en = MAPS.ME uses your data to improve your experience. Changes will take effect after restart the app.
```

Comment:
en = We may use your data to improve MAPS.ME experience. Changes will take effect after you restart the app.